pkg/chezmoi/archive_test.go

```diff
@@ -11,51 +11,76 @@ import (
 	"github.com/twpayne/chezmoi/v2/pkg/archivetest"
 )
 
+var nestedRoot = map[string]any{
+	"dir1": map[string]any{
+		"subdir1": map[string]any{
+			"file1": "",
+			"file2": "",
+		},
+		"subdir2": map[string]any{
+			"file1": "",
+			"file2": "",
+		},
+	},
+	"dir2": map[string]any{
+		"subdir1": map[string]any{
+			"file1": "",
+			"file2": "",
+		},
+		"subdir2": map[string]any{
+			"file1": "",
+			"file2": "",
+		},
+	},
+	"file1":    "",
+	"file2":    "",
+	"symlink1": &archivetest.Symlink{Target: "file1"},
+	"symlink2": &archivetest.Symlink{Target: "file2"},
+}
+
+var flatRoot = map[string]any{
+	"dir1/subdir1/file1": "",
+	"dir1/subdir1/file2": "",
+	"dir1/subdir2/file1": "",
+	"dir1/subdir2/file2": "",
+	"dir2/subdir1/file1": "",
+	"dir2/subdir1/file2": "",
+	"dir2/subdir2/file1": "",
+	"dir2/subdir2/file2": "",
+	"file1":              "",
+	"file2":              "",
+	"symlink1":           &archivetest.Symlink{Target: "file1"},
+	"symlink2":           &archivetest.Symlink{Target: "file2"},
+}
```

Comment:
These two variables can be moved inside the `TestWalkArchive` function as they're only used in that function and so don't need to be global.